### ISSUE TITLE
Is "set protocols ospf area 0.0.0.0 export opsf-export" invalid?

### ISSUE THREAD

**Issue by junpei-yoshino** [codeout/junoser#6]
In version 0.1.5, "set protocols ospf area 0.0.0.0 export opsf-export" is valid.
"set protocols ospf export opsf-export" must be valid.
Not in "area 0.0.0.0".
There is something wrong.

**Comment by codeout** [comment]
Should be a bug - "set protocols ospf area 0.0.0.0 export opsf-export" must be invalid according to JUNOS CLI.
I'll take a look into the parser and XML schema this weekend.

**Comment by codeout** [comment]
This is a sequence-of-choice problem.
JUNOS CLI sometimes accept one-line configuration, sometimes reject. Unfortunately .xsd doesn't help.

```
set system syslog archive size 1000k files 10 world-readable  # valid
set protocols ospf area 0.0.0.0 <snip> export ospf-export     # invalid
```

against the similar xsd:

``` xml
<xsd:sequence><xsd:choice/><xsd:sequence>
```

In case of `protocol ospf`, we should hack parser.rb which is translated from .xsd to reject the sequence-of-choice.

**Comment by codeout** [comment]
It's fixed at 0.2.0. Thank you for your report to junoser! :snowman: 

```
$ cat config
set protocols ospf area 0.0.0.0 export opsf-export
$ junoser -v
0.2.0
$ junoser -c config
Invalid syntax:  set protocols ospf area 0.0.0.0 export opsf-export
zsh: exit 1     junoser -c config
```

In my benchmark, 0.2.0 is 150% faster than 0.1.x because of AST optimization.
